src/components/TimelineNav/TimelineNav.tsx

```diff
@@ -334,15 +334,31 @@ export const TimelineNav = ({
   };
 
   /**
-   * onClick function
-   * 1) Triggered by 'Back' button on < lg viewports
-   * 2) Return focus to the button that was clicked to open the current panel; activeIndexState holds the index of the last nav item selected in the timelinNavItemRefs array
-   * 3) Body panel is visible/hidden depending on true/false value of isActive; hide it by setting isActive to false
+   * onClick (used by 'Back' button on < lg viewports)
+   * 1) Return focus to the button that was clicked to open the current panel; activeIndexState holds the index of the last nav item selected in the timelinNavItemRefs array
+   * 2) Body panel is visible/hidden depending on true/false value of isActive; hide it by setting isActive to false
    */
   const onClick = () => {
-    timelineNavItemRefs[activeIndexState].current?.focus(); /* 2 */
-    setIsActive(false); /* 3 */
+    timelineNavItemRefs[activeIndexState].current?.focus(); /* 1 */
+    setIsActive(false); /* 2 */
   };
+
+  /**
+   * resetTabIndex (used by 'Back' button on <lg viewports)
+   * 1) This fixes an issue where a keyboard user who has focus on the
+   * 'Back' button could use shift-tab to put keyboard focus back
+   * on the menu, which is currently off-canvas. To prevent this
+   * unwanted behavior, we set the currently active menu item's
+   * tabIndex to "-1" so that it can't be focusable in sequential
+   * keyboard navigation. The 'Back' button should be the only way
+   * users are able to return to the nav list, both visually and
+   * via keyboard navigation.
+   * See https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/tabindex#sect1
+   */
+  const resetTabIndex = () => {
+    timelineNavItemRefs[activeIndexState].current.tabIndex = '-1';
```

Comment:
Do we need to change it back at some point? Testing it out, after I go back to the timeline, if I tab away, I can't get back to the timeline via keyboard.

Reply:
Great catch, I missed that. I added a line to the Back button's onClick to replace the current nav item in the tab order. In the tests I tried with this fix, I'm always able to move in and out of the timeline with tab/shift-tab.